smslibrary/src/test/java/com/eis/smslibrary/SMSManagerTest.java

```diff
@@ -25,7 +30,13 @@
     private final String SENT_TEXT = (char) 0x02 + MESSAGE_TEXT;
     private final SMSPeer VALID_PEER = new SMSPeer(PEER_TEXT);
     private final SMSMessage VALID_MESSAGE = new SMSMessage(VALID_PEER, MESSAGE_TEXT);
-    private SMSManager instance = SMSManager.getInstance();
+    private SMSManager smsManager = SMSManager.getInstance();
+
+    @Before
+    public void setup(){
+        PowerMockito.mockStatic(SmsManager.class);
+        when(SmsManager.getDefault()).thenReturn(managerMock);
+    }
```

Comment:
I'll copy this in #20 , `nullMessage_throws()` was working even without PowerMockito, but now I see that the `NullPointerException` was being thrown for the wrong reason.